OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVREntryPoint.cpp

```diff
@@ -29,13 +29,33 @@ DEFINE_LOG_CATEGORY(OSVREntryPointLog);
 
 OSVREntryPoint::OSVREntryPoint()
 {
+}
+
+void OSVREntryPoint::Initialize()
+{
+    // avoid double initialization
+    if (osvrClientContext != nullptr)
+    {
+        // too spammy to log here
+        return;
+    }
+
+    // hack: to avoid delay on OSVR server connection attempt when not using OSVR, don't initialize if nohmd command line param specified
+    if (FParse::Param(FCommandLine::Get(), TEXT("nohmd")))

```

Comment:
One problem: if OSVR plugin is enabled but running on a non-VR machine, then when OSVRInput initializes (which happens even if nohmd passed) the 3-10 second pause would occur due to not being able to connect to OSVR server.